### PR TITLE
[pota-value-test] Generate input h5 from circle

### PR DESCRIPTION
**Pull request by jinevening**
This generates input h5 file from circle model

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Comment by jinevening** [comment]
@mhs4670go , This test requires 'explicit' inputs given by text files, so I could not use `testDataGenerator`.

File: compiler/pota-quantization-value-test/CMakeLists.txt [hunk 1]
```diff
@@ -14,8 +20,11 @@ include("test.local.lst" OPTIONAL)
 unset(TEST_DEPS)
 
 get_target_property(ARTIFACTS_BIN_PATH testDataGenerator BINARY_DIR)
+get_target_property(SCHEMA_BIN_PATH mio_circle BINARY_DIR)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/gen_h5_explicit_inputs.py" "${CMAKE_CURRENT_BINARY_DIR}/gen_h5_explicit_inputs.py" COPYONLY)
```
Comment (seanshpark) [inline compiler/pota-quantization-value-test/CMakeLists.txt:25]: Can you please split this line so that max column is 80 (good) or 100 ?

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**